### PR TITLE
document access control

### LINKED ISSUE #1184: Document the high level RBAC model
### Related code PR

_No response_

### Which part(s) of the docs might be affected or should be updated? And how?

The draft:

https://www.notion.so/risingwave-labs/Role-Based-Access-Control-RBAC-Guide-Draft-d38d8c47000048e7baa3399a323b39b2?pvs=4

### Reference

_No response_

### PR DESCRIPTION
**Pull request by hengm3467**
## Info

- **Description**

  Document access control in RisingWave.

- **Notes**

  [ Include any supplementary context or references here. ]

- **Related code PR**

  [ Provide a link to the relevant code PR here, if applicable. ]

- **Related doc issue**
  
  Resolves #1184

## For reviewers

- **Preview**

  [ Paste the preview link to the updated page(s) here. Edit this item after the preview site is ready. To find the updated pages, scroll down to locate and open the Amplify preview link and select the **upcoming** version of the documentation. ]

- **Key points**

  [ Parts that may need revision or extra consideration. ]

## Before merging

- [ ] I have checked the doc site preview, and the updated parts look good.

- [ ] I have acquired the approval from the owner (and optionally the reviewers) of the code PR and at least one tech writer (`CharlieSYH`, `emile-00`, & `hengm3467`).

### CODE REVIEW

File: docs/manage/access-control.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+```
+
+### Alter user
+
+You can alter the initial permissions and the password of a user by using the `ALTER USER` command.
+
+The following statement modifies the password and initial permissions of `user001`.
+
+```sql
+ALTER USER user001 NOSUPERUSER CREATEDB PASSWORD '4d2Df1ee5';
```
Comment (yezizp2012) [inline docs/manage/access-control.md]:
```suggestion
ALTER USER user001 WITH NOSUPERUSER CREATEDB PASSWORD '4d2Df1ee5';
```

Comment (yezizp2012) [reply docs/manage/access-control.md]: We also support rename operation for each user, we can also add it here. the syntax is like:
```sql
ALTER USER user RENAME TO user1;
```

**Review by yezizp2012 — COMMENTED** [review]
Rest LGTM!

File: docs/manage/access-control.md [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+
+When creating a user, the administrator of an organization can determine the system-level permissions and set a password. The system permissions and the user names can be revised with the `ALTER USER` command. For details about the system permissions, see [System permissions](/sql/commands/sql-create-user.md#system-permissions).
+
+Database privileges can be configured later by using `GRANT` and `REVOKE` commands. The privileges are managed at these object levels:
+
+- Database
+- Schema
+- Table
+- Source
+- Materialized view
```
Comment (neverchanje) [inline docs/manage/access-control.md:17]: and Sink

**Review by neverchanje — APPROVED** [review]
Rest LGTM. This is in a very good quality.

File: docs/manage/access-control.md (hunk 2 above)
Comment (hengm3467) [reply docs/manage/access-control.md:17]: @neverchanje It looks like no actions have been defined for sinks yet (confirmed with @yezizp2012), so we can ignore sinks for now.

https://github.com/risingwavelabs/risingwave/blob/5484b07f7ec35a1e3f9bc4516128a8959892e341/src/frontend/src/user/user_privilege.rs#L36

Comment (neverchanje) [reply docs/manage/access-control.md:17]: Thanks Heng. It'll be in our plan.

**Pull request merged**